client/query/query.go

```diff
@@ -2,20 +2,21 @@ package query
 
 import (
 	"github.com/binance-chain/go-sdk/client/basic"
+	"github.com/binance-chain/go-sdk/common/types"
 )
 
 type QueryClient interface {
 	GetClosedOrders(query *ClosedOrdersQuery) (*CloseOrders, error)
```

Comment:
Moved to types package. But since same api and rpc sometimes get different result data structure, I put it in same packages, a bit confusing for user.